easybuild/easyconfigs/p/PyTorch-Geometric/PyTorch-Geometric-2.0.1-fosscuda-2020b-PyTorch-1.9.0.eb

```diff
@@ -54,6 +54,13 @@ exts_list = [
     }),
 ]
 
+preinstallopts = 'export TORCH_CUDA_ARCH_LIST="%(cuda_cc_semicolon_sep)s" ; '
```

Comment:
What if EasyBuild is not configured with `--cuda-compute-capabilities`, then this will be an empty value...

Reply:
Then we're back to zero, and there won't gpu support, right?

Reply:
`$TORCH_CUDA_ARCH_LIST` being defined but empty or not being defined is not the same thing (but I haven't tested this, I must admit)